webpages/popup/index.html

```diff
@@ -8,6 +8,7 @@
       @import url("../styles/components/tooltips.css");
       @import url("../styles/colors.css");
     </style>
+    <link rel="icon" href="../../images/icon.png" />
```

Comment:
Users aren't expected to open this file as a tab, so this looks unnecessary.

Reply:
Doesn't really matter